libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/model/ProjectModel.kt

```diff
@@ -10,6 +10,15 @@ import java.io.File
 @Serializable
 class ProjectModel(
     val serviceNames: Set<String>,
+
+    // Telemetry
+    val gradleVersion: String?,
+    val androidMinSdk: Int?,
+    val androidTargetSdk: Int?,
+    val androidCompileSdk: String?,
+    val androidAgpVersion: String?,
+    val apolloGenerateSourcesDuringGradleSync: Boolean?,
+    val apolloLinkSqlite: Boolean?,
```

Comment:
That doesn't belong to the compiler. `apollo-compiler` shouldn't need to care about Gradle and/or Android.

That's probably an argument to remove `ProjectModel` altogether. Maybe it can keep using the tooling API. Or if this is too slow, a separate task that generates generic Gradle info. If you want to keep the typesafe class, you'll have to publish an intermediate artifact that both `intellij-plugin` and `apollo-gradle-plugin` depend on. 
But you can also just dump a bunch of unstructured JSON from `apollo-gradle-plugin` and document the structure when reading it in `intellij-plugin`.

Reply:
> That doesn't belong to the compiler. apollo-compiler shouldn't need to care about Gradle and/or Android.

Hmm good point.

I still need something like a `ProjectModel` to get the list of services though. Maybe just a JSON list is enough, without needing an actual model...

I could move these to `apollo-tooling`, it's already a dependency of both the IJ plugin and `apollo-gradle-plugin-tasks`, plus "tooling" is on point. Wdyt?